docs/config.json

```diff
@@ -709,6 +709,10 @@
               "title": "MongoDB Atlas",
               "slug": "/database-access/guides/mongodb-atlas/"
             },
+            {
+              "title": "Self-Hosted Elastic",
```

Comment:
Elastic or Elasticsearch? Is there a difference or do we just use them interchangeably?

Reply:
AFAIK `Elasticsearch` is the canonical name we should use, Elastic is ambiguous and possibly confusing.

Reply:
```suggestion
              "title": "Self-Hosted Elasticsearch",
```